Canvas Added to Now Playing View

egrinant opened the pull request:
Canvas is now playable on the new "Now Playing" view. The changes made to the code base are minimal.

The player is cleared and created again when switching between full screen and windowed mode.

Small quality change to the code by replacing `querySelectorAll` with `querySelector`, as only the first element was being used.

egrinant committed:
Canvas Added to Now Playing View

itsmeow commented:
If you could update the README accordingly as well, it does mention the fullscreen view...

itsmeow commented:
Maybe a switch (boolean flag) at the top to switch between the two modes would be ideal

itsmeow commented:
Oh wait I see, both are supported. A flag to disable either, then

egrinant committed:
Ability to select which views will be rendered

egrinant commented:
@itsmeow I added the config object where you can decide which view you want, both are enabled by default. I also updated the readme file with the configuration topic.

itsmeow reviewed (approved):
Looks great, thank you so much!
